test/helper/index.js

```diff
@@ -118,7 +118,13 @@ export function bootstrapBpmnJS(BpmnJS, diagram, options, locals) {
 
     setBpmnJS(instance);
 
-    instance.importXML(diagram, done);
+    return new Promise(function(resolve) {
+      instance.importXML(diagram).then(function(result) {
+        resolve({ error: null, warnings: result.warnings });
+      }).catch(function(err) {
+        resolve({ error: err, warnings: err.warnings });
+      });
+    });
```

Comment:
How about if we do chaining here?

```suggestion
    return instance.importXML(diagram)
      .then(function(result) {
        return({ error: null, warnings: result.warnings });
      })
      .catch(function(err) {
        resolve({ error: err, warnings: err.warnings });
      });
```